### PR TITLE
Button/Popup for users to make edits to their bookings

### LINKED ISSUE #32: Editing existing bookings
**User Story:**

As a user I want to be able to edit bookings after I have already made them so I can cancel or change them

**Acceptance Criteria:**

[ ] Can enter booking number information to application which takes user to booking setup page

**Additional Context:**
Does not actually have to work just yet (i.e. not use dummy data) - API has not been setup linking booking number to a restaurant

### PR DESCRIPTION
**Pull request by saddboys**
## Link to Issue Number
closes #32 

## Description
Created a button/popup which would allow the user to edit existing bookings. This currently has no functionality (uses dummy data), but can be changed to use actual data when API is online. It takes in the booking ID number, and allows a user to change the details of that booking.

## Checklist
- [x] All feature request A/C have been met OR the bug has been fixed
- [x] I ran all nessisary tests and they pass
- [x] I rebased upstream/master onto my working branch before opening this PR
- [x] I have named my PR something sensible
- [x] I have included the issue number above
- [x] I have assigned at least two people to review my changes

## Other Notes
Test(s) ran had complete coverage - until further tests became trivial
The popup has functionality but basic aesthetic design, awaiting on further confirmation for design.

### CODE REVIEW

File: frontend/src/booking/BookingEditPopupButton.js [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+        changePath("/booking", history);
+    };
+
+
+    return (
+        (isInput) ?
+        <Dialog onClose={handleClosePopup} transitionDuration={0} aria-labelledby="simple-dialog-title" open={open}>
+
+            <DialogTitle id="simple-dialog-title">Input Booking ID</DialogTitle>
+            <TextField id="outlined-basic" label="Insert ID here" variant="outlined" />
```
Comment (Sabatr) [inline frontend/src/booking/BookingEditPopupButton.js]: There should be a check whether the user has entered something before clicking confirm.

**Comment by saddboys** [comment]
Above comments noted and fixed

File: frontend/src/booking/BookingEditPopupDialog.js [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+    IDSwitchMethod(dummyBooking.name);
+    changePath("/booking", history);
+  };
+
+  return (
+    (isInput)
+      ? (
+        <Dialog onClose={handleClosePopup} transitionDuration={0} aria-labelledby="simple-dialog-title" open={open}>
+          <div>
+            <DialogTitle id="simple-dialog-title">Input Booking ID</DialogTitle>
```
Comment (Sabatr) [inline frontend/src/booking/BookingEditPopupDialog.js]: Missed one

**Comment by saddboys** [comment]
Changing PR to pull from feature branch instead of master! Closed

**Pull request closed without merging**